test/test_client.py

```diff
@@ -1,13 +1,22 @@
 # These tests test the behaviours expected of wsproto in when the
-# connection is a client.
+# connectionis a client.
+from typing import List, Optional, Tuple
 
 import h11
 import pytest
 
 from wsproto import WSConnection
 from wsproto.connection import CLIENT
-from wsproto.events import AcceptConnection, RejectConnection, RejectData, Request
+from wsproto.events import (
+    AcceptConnection,
+    Event,
+    RejectConnection,
+    RejectData,
+    Request,
+)
```

Comment:
Is this our new coding style? I kinda prefer the single line without parenthesis - unless you argue against it?

Reply:
This is now `black`'s  choice, and this is what it does...